make.bat

```diff
@@ -26,19 +26,21 @@ choice /c:1234567QpbD>NUL
 :: Debug = OFF [default]
 
 :: OPTIONS
-if errorlevel 11 goto debug
-if errorlevel 10 goto WinBmi2
-if errorlevel 9 goto WinPopcnt
-if errorlevel 8 goto done
-if errorlevel 7 goto matefinder
-if errorlevel 6 goto base
-if errorlevel 5 goto arm
-if errorlevel 4 goto mac
-if errorlevel 3 goto linux
-if errorlevel 2 goto windows
 if errorlevel 1 goto allBinaries
+if errorlevel 2 goto windows
+if errorlevel 3 goto linux
+if errorlevel 4 goto mac
+if errorlevel 5 goto arm
+if errorlevel 6 goto base
+if errorlevel 7 goto matefinder
+if errorlevel 8 goto done
+if errorlevel 9 goto WinPopcnt
+if errorlevel 10 goto WinBmi2
+if errorlevel 11 goto debug
+) else (
 echo CHOICE missing
-goto done
```

Comment:
Again, this is intentional. Remember: if something isn't broken, don't fix it.

Reply:
I saw that in a previous commit you removed a `:done`.

Reply:
@Counterply, putting %errorlevel% instead of errorlevel may solve the problem.

Reply:
Is it ok, now?